src/SprykerSdk/Glue/Benchmark/FactoryResolver/FactoryResolver.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Glue\Benchmark\FactoryResolver;
+
+use Spryker\Glue\Kernel\ClassResolver\Factory\FactoryResolver as KernelFactoryResolver;
```

Comment:
Why using an alias?

Reply:
PHPStorm was complaining on the same class definition without it